src/QsCompiler/Core/Dependencies.fs

```diff
@@ -87,192 +87,89 @@ type BuiltIn = {
-                InferredInformation = InferredCallableInformation.NoInformation
-            }
-
-        let args = 
-            [
-                Result |> ResolvedType.New;
-                [
-                    (
-                        (
-                            typeParamT,
-                            UnitType |> ResolvedType.New
-                        ),
-                        characteristics
-                    ) |> Operation |> ResolvedType.New;
-                    typeParamT
-                ].ToImmutableArray() |> TupleType |> ResolvedType.New;
-                [
-                    (
-                        (
-                            typeParamU,
-                            UnitType |> ResolvedType.New
-                        ),
-                        characteristics
-                    ) |> Operation |> ResolvedType.New;
-                    typeParamU
-                ].ToImmutableArray() |> TupleType |> ResolvedType.New
-            ].ToImmutableArray() |> TupleType |> ResolvedType.New
-
-        (
-            (
-                args,
-                ResolvedType.New UnitType
-            ),
-            characteristics
-        ) |> Operation |> ResolvedType.New
 
     // This is expected to have type <'T, 'U>((Result, (('T => Unit), 'T), (('U => Unit), 'U)) => Unit)
     static member ApplyIfElseR = {
         Name = "ApplyIfElseR" |> NonNullable<string>.New
         Namespace = BuiltIn.ClassicallyControlledNamespace
         TypeParameters = ImmutableArray.Create("T" |> NonNullable<string>.New, "U" |> NonNullable<string>.New)
     }
-    static member ApplyIfElseRResolvedType = BuiltIn._MakeApplyIfElseResolvedType BuiltIn.ApplyIfElseR []
 
     // This is expected to have type <'T, 'U>((Result, (('T => Unit is Adj), 'T), (('U => Unit is Adj), 'U)) => Unit is Adj)
     static member ApplyIfElseRA = {
         Name = "ApplyIfElseRA" |> NonNullable<string>.New
         Namespace = BuiltIn.ClassicallyControlledNamespace
         TypeParameters = ImmutableArray.Create("T" |> NonNullable<string>.New, "U" |> NonNullable<string>.New)
     }
-    static member ApplyIfElseRAResolvedType = BuiltIn._MakeApplyIfElseResolvedType BuiltIn.ApplyIfElseRA [OpProperty.Adjointable]
 
     // This is expected to have type <'T, 'U>((Result, (('T => Unit is Ctl), 'T), (('U => Unit is Ctl), 'U)) => Unit is Ctl)
     static member ApplyIfElseRC = {
         Name = "ApplyIfElseRC" |> NonNullable<string>.New
         Namespace = BuiltIn.ClassicallyControlledNamespace
         TypeParameters = ImmutableArray.Create("T" |> NonNullable<string>.New, "U" |> NonNullable<string>.New)
     }
-    static member ApplyIfElseRCResolvedType = BuiltIn._MakeApplyIfElseResolvedType BuiltIn.ApplyIfElseRC [OpProperty.Controllable]
 
     // This is expected to have type <'T, 'U>((Result, (('T => Unit is Adj + Ctl), 'T), (('U => Unit is Adj + Ctl), 'U)) => Unit is Adj + Ctl)
     static member ApplyIfElseCA = {
```

Comment:
Regarding the naming, please follow and match this PR: https://github.com/microsoft/qsharp-runtime/pull/139